daemon/src/projection.rs

```diff
@@ -494,6 +494,7 @@ impl Cfd {
                 self.aggregated.timelocked_cet = Some(timelocked_cet);
                 self.closing_price = Some(price);
 
+                self.aggregated.commit_published = true;
```

Comment:
👍 I think we can add this to `RefundTimelockExpired` too. Also should it also be part of the `CetTimelockExpiredPostOracleAttestation` and `CetTimelockExpiredPriorOracleAttestation`?

Reply:
I would only set it in scenarios that are *directly* related to publishing `commit`. That is at least what we have been doing so far. 
`RefundTimelockExpired` sets `self.aggregated.refund_published = true;` because it is about the refund tx and only indirectly implies `commit`. But `RefundTimelockExpired` implies `CommitConfirmed` which sets `self.aggregated.commit_published = true;`.

I thought about `CetTimelockExpiredPostOracleAttestation` and `CetTimelockExpiredPriorOracleAttestation` as well, but they also only apply things, but don't direcly relate to publishing `commit` - they imply `CommitConfirmed`. 
For me publishing `commit` is only directly related to `ManualCommit`, `OracleAttestedPriorCetTimelock` (in a case where `ManualCommit` is not triggered). Additionally `CommitConfirmed` is relevant for cases where the other party published `commit`.
Note that `OracleAttestedPostCetTimelock` implies that `CommitConfirmed` as well.

Reply:
We *could* opt for covering cases less strict, but that will make it hard to reason about the code I feel.

Reply:
I'm okay with your reasoning, so no need to change anything.

I find this whole `match` statement hard to follow, but I don't have a better idea a this point.